docs/api-docs/api-docs.yml

```diff
@@ -195,27 +195,50 @@ paths:
               example:
                 [ "hash", "mask", "redact", "replace", "encrypt" ]
 
-  /decrypt:
+  /deanonymizers:
```

Comment:
something is wrong with the file structure:
https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/presidio/faf66c7364f6d945730a1a1919fc1df7628ed6e9/docs/api-docs/api-docs.yml
I think that DeanonymizeRequest is defined in 'components', not 'requestBodies'

Reply:
I'll check it out. tnx.

Reply:
Done.